responsive-grid-layout/dilara/global.css

```diff
@@ -11,18 +11,27 @@ body {
 
 .header {
   background-color: #006c67;
+  display: grid;
+  grid-area: header;
 }
 .sidebar-left {
   background-color: #003844;
+  grid-area: sidebar-left;
 }
+
 .sidebar-right {
   background-color: #ffb100;
+  grid-area: sidebar-right;
 }
 .content {
   background-color: #ffebc6;
+  grid-area: content;
+  
 }
 .footer {
   background-color: #006c67;
+  grid-area: footer;
+  height: 200px;
```

Comment:
I'm pretty sure that you don't need to set the height of the footer explicitly to 200px here. Since you don't change the default alignment (stretch), it will take as much space as you specify for the last row (300px in this case).